pkgs/build-support/rust/build-rust-package/default.nix

```diff
@@ -134,6 +131,7 @@ stdenv.mkDerivation ((removeAttrs args [ "depsExtraArgs" "cargoUpdateHook" "carg
   ];
 
   buildInputs = buildInputs
+    ++ [ cacert ] # FIXME Is this actually needed?
```

Comment:
Does this even have an effect when its in buildInputs?

Reply:
Yes:
```
$ nix print-dev-env --impure --expr 'with import ./. {}; pkgsCross.riscv64.mkShell { buildInputs = [ cacert ]; }' | grep SSL
NIX_SSL_CERT_FILE='/nix/store/65wwvlzss9wsrsjwidrp2i0p0ps38nvg-nss-cacert-3.83/etc/ssl/certs/ca-bundle.crt'
export NIX_SSL_CERT_FILE

$ nix print-dev-env --impure --expr 'with import ./. {}; pkgsCross.riscv64.mkShell { }' | grep SSL

$
```

Reply:
@Mic92 asked the other way around, what should having it in nativeBuildInputs achieve?

AFAIK, the only distinction made is when cross-compiling where nativeBuildInputs are expected to run on the host while buildInputs are for the target.

Since you're here, do you know whether some rust packages need this env var to build a usable package? (Especially w.r.t. internet-facing ones.)